src/pages/docs/tutorials/tutorial-hello-enact/basics/index.md

```diff
@@ -100,6 +100,9 @@ return (
 ```
 You'll notice that by introducing `<div>` we no longer have valid JavaScript! In fact, it looks a lot like valid HTML. That's because React introduces [JSX](https://reactjs.org/docs/jsx-in-depth.html), which is a JavaScript syntax extension. In order to make JSX runnable by the browser, it has to be converted to JavaScript. With Enact, this is handled during the build process using [webpack](https://webpack.js.org) and [babel](https://babeljs.io/). More on this [later](#running-the-app).
 
+You may wonder why import `React` even if it's not explicitly used. That is because it's used by transpiled JavaScript which was JSX. Thus, importing `React` was necessary to use JSX. After React 17, these are [improved](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#whats-different-in-the-new-transform), so no longer needed to import it. You can remove the code (`import React from 'react'`) since our app uses React 17.
```

Comment:
Back in the time, we need to import `react` module in order to use JSX. But as of now, we no longer needed to import since React 17 introduced [the new JSX transform](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html).

Reply:
Thank you :)